src/layouts/Base.astro

```diff
@@ -9,6 +9,11 @@ export interface Props {
 
 const { title, skipTargetId } = Astro.props as Props;
 
+const canonicalURL = new URL(
+	Astro.url.pathname.replace(/([^/])$/, '$1/'),
```

Comment:
What does this do? Looks like it ensures there's always a trailing slash? If a trailing slash existed, wouldn't it result in 2 trailing slashes? Is it guaranteed this value will never have a trailing slash?

Reply:
I just did a couple tests. This replacement ensures that there is always exactly 1 trailing slash on the url, and that the canonical url of a given page always reflects its full url.

For example, the index page of the site will be `https://the-collab-lab.codes/` but the canonical url of the volunteer page will be `https://the-collab-lab.codes/volunteer/`

Reply:
Excellent!